swift/ios-shoppe-demo/Views/LoginButtonTableViewCell.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  LoginButtonTableViewCell.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/19/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import UIKit
+
+class LoginButtonTableViewCell: UITableViewCell {
+
+    var loginViewController: LoginCollectionViewController?
```

Comment:
If this cell is owned by `LoginCollectionViewController` this will cause a strong reference cycle and thus leak memory. That can be fixed by making this a `weak var`.

It's more idiomatic to make a delegate protocol for this type to handle `login`, but that might be over-engineering it.